pkg/cloudprovider/providers/aws/sets_ippermissions.go

```diff
@@ -59,7 +59,7 @@ func (s IPPermissionSet) Ungroup() IPPermissionSet {
 			c := &ec2.IpPermission{}
 			*c = *p
 			c.UserIdGroupPairs = []*ec2.UserIdGroupPair{u}
-			l2 = append(l, c)
+			l2 = append(l2, c)
```

Comment:
Good catch :+1:

Reply:
I don't know what the implications of this bug are, but this seems markedly more severe. should this be split out to backport?

Reply:
did we ever fix this?